docs/en/04_dataobjects.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Using DataObjects with Fluent
+summary: Configuring DataObjects for use with Fluent, including CMS fields, extensions, and versioned objects.
+icon: database
+---
+
+# Using dataObjects with fluent
+
+## CMS fields
+
+With each of the possible extensions that come with Fluent, it's necessary to configure your DataObjects to
+correctly delegate your `$fields` to extensions. A different approach may be needed for each DataObject, depending
+on how it generates it's fields.
+
+In all cases, it is necessary to ensure that the `updateCMSFields()` extension method is called once (and
+only once) on each `FieldList` object.
+
+### Default field scaffolder
+
+If using a default field scaffolder (such as [`SiteTree::getCMSFields()`](api:SilverStripe\CMS\Model\SiteTree::getCMSFields()) or `DataObject::getCMSFields`)
```

Comment:
```suggestion
If using a default field scaffolder (such as [`DataObject::getCMSFields()`](api:SilverStripe\ORM\DataObject::getCMSFields()))
```
It's the same method, so only should be mentioned once (especially if we're only linking to one)

Reply:
Updated